core/src/trezor/lvglui/scrs/address.py

```
@@ -513,25 +624,35 @@
         msg_class = getattr(messages, self.current_chain_info["msg_class"])
         return msg_class(address_n=path, show_display=False)
 
-    def _prepare_chain_message(self, msg_type):
+    def _prepare_chain_message(self, name):
```

Ruff (0.8.2)

`_prepare_chain_message` is too complex (12 > 10)

(C901)